art/utils.py

```diff
@@ -391,7 +391,7 @@ def tanh_to_original(
 # --------------------------------------------------------------------------------------------------- LABELS OPERATIONS
 
 
-def to_categorical(labels: np.ndarray, nb_classes: Optional[int] = None) -> np.ndarray:
+def to_categorical(labels: Union[np.ndarray, List[float]], nb_classes: Optional[int] = None) -> np.ndarray:
```

Comment:
Why is the list of floats type required? Could we instead convert potential list input into a ndarray?

Reply:
The conversion happens in the first line of the function, so the function already works just as well for lists of floats and np arrays. I was just adding the type, so lists could also be accepted.